### PR TITLE
Fix bugs and minor refactor in vortran laser class

### PR DESCRIPTION
**Pull request by keithchev**
I noticed a few typos/bugs in the newly merged vortran laser class. I also slightly refactored the `get_lasers` method for clarity. 

There may also be some missing logic at line 419 (where I changed a no-op to `pass`).

### CODE REVIEW

**Review by AhmetCanSolak — APPROVED** [review]
To me it looks good, but I would appreciate if we can merge this after @edyoshikun 's review and after merging #154

**Review by edyoshikun — APPROVED** [review]
LGTM! I testd it in person and works. Thank you both.

**Pull request merged**
